# Translations template for fb_vmware.
# Copyright (C) 2024 Frank Brehm, Berlin
# This file is distributed under the same license as the fb_vmware project.
# Frank Brehm <frank.brehm@pixelpark.com>, 2024.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: fb_vmware 1.3.0\n"
"Report-Msgid-Bugs-To: frank.brehm@pixelpark.com\n"
"POT-Creation-Date: 2024-07-25 17:19+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <frank.brehm@pixelpark.com>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.15.0\n"

#: bin/get-vsphere-host-list:65 bin/get-vsphere-storage-cluster-list:65 bin/get-vsphere-vm-info:65
#: bin/get-vsphere-vm-list:65
msgid ""
"{c}-Object:\n"
"{a}"
msgstr ""

#: lib/fb_vmware/about.py:283 lib/fb_vmware/cluster.py:368 lib/fb_vmware/controller.py:292
#: lib/fb_vmware/datastore.py:230 lib/fb_vmware/dc.py:183 lib/fb_vmware/disk.py:388
#: lib/fb_vmware/ds_cluster.py:129 lib/fb_vmware/ether.py:510 lib/fb_vmware/host.py:247
#: lib/fb_vmware/host.py:670 lib/fb_vmware/host_port_group.py:198 lib/fb_vmware/network.py:141
#: lib/fb_vmware/vm.py:640
msgid "The given parameter {p!r} on calling method {m}() has failing attributes"
msgstr ""

#: lib/fb_vmware/about.py:291 lib/fb_vmware/cluster.py:279 lib/fb_vmware/dc.py:191
#: lib/fb_vmware/disk.py:397
msgid "Parameter {t!r} must be a {e} object, a {v} object was given instead."
msgstr ""

#: lib/fb_vmware/about.py:304 lib/fb_vmware/cluster.py:304 lib/fb_vmware/controller.py:334
#: lib/fb_vmware/datastore.py:270 lib/fb_vmware/dc.py:219 lib/fb_vmware/disk.py:419
#: lib/fb_vmware/ds_cluster.py:155 lib/fb_vmware/ether.py:475 lib/fb_vmware/host.py:279
#: lib/fb_vmware/host.py:588 lib/fb_vmware/host_port_group.py:222 lib/fb_vmware/network.py:173
#: lib/fb_vmware/vm.py:509
msgid "Creating {} object from:"
msgstr ""

#: lib/fb_vmware/about.py:340 lib/fb_vmware/controller.py:339 lib/fb_vmware/disk.py:424
#: lib/fb_vmware/ether.py:480 lib/fb_vmware/host_port_group.py:227 lib/fb_vmware/vm.py:590
msgid "Created {} object:"
msgstr ""

#: lib/fb_vmware/app/__init__.py:98
msgid "{what} of {app} ..."
msgstr ""

#: lib/fb_vmware/app/__init__.py:101
msgid "Did not found any configured Vsphere environments."
msgstr ""

#: lib/fb_vmware/app/__init__.py:109
msgid "Checking for configured VSPhere instance {!r} ..."
msgstr ""

#: lib/fb_vmware/app/__init__.py:113
msgid "VSPhere {!r} not found in list of configured VSPhere instances."
msgstr ""

#: lib/fb_vmware/app/__init__.py:138
msgid "The VSPhere names from configuration, in which the VMs should be searched."
msgstr ""

#: lib/fb_vmware/app/__init__.py:146
msgid "Got command line arguments:"
msgstr ""

#: lib/fb_vmware/app/__init__.py:152
msgid "Initializing VSphere handlers ..."
msgstr ""

#: lib/fb_vmware/app/__init__.py:165
msgid "Initializing handler for VSPhere {!r} ..."
msgstr ""

#: lib/fb_vmware/app/__init__.py:178
msgid "Could not initialize {} object from:"
msgstr ""

#: lib/fb_vmware/app/__init__.py:188
msgid "Cleaning up ..."
msgstr ""

#: lib/fb_vmware/app/__init__.py:192
msgid "Closing VSPhere object {!r} ..."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:58
msgid "Tries to get a list of all physical hosts in VMWare VSphere and print it out."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:102 lib/fb_vmware/app/get_vm_list.py:131
msgid "Filter options"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:107
msgid "for names of hosts"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:108
msgid "A regular expression to filter the output list of hosts by their name (Default: {!r})."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:116
msgid "Filter output for online hosts."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:120
msgid "Filter output for offline hosts and templates."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:123 lib/fb_vmware/app/get_storage_cluster_list.py:103
#: lib/fb_vmware/app/get_vm_list.py:177
msgid "Output options"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:127 lib/fb_vmware/app/get_storage_cluster_list.py:112
#: lib/fb_vmware/app/get_vm_list.py:186
msgid ""
"The keys for sorting the output. Available keys are: {avail}. The default sorting keys are: "
"{default}."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:142 lib/fb_vmware/app/get_vm_list.py:204
msgid "Regular expression for filtering: {!r}"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:145 lib/fb_vmware/app/get_vm_list.py:207
msgid "Got a {c} for pattern {p!r}: {e}"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:155 lib/fb_vmware/app/get_storage_cluster_list.py:133
#: lib/fb_vmware/app/get_vm_info.py:82 lib/fb_vmware/app/get_vm_list.py:243
msgid "Starting {a!r}, version {v!r} ..."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:179
msgid "Getting all VSPhere hosts ..."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:193
msgid "First found host:"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:250
msgid "Connect state"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:251
msgid "CPU cores/threads"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:252
msgid "Memory in GiB"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:253
msgid "Vendor"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:254
msgid "Model"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:255
msgid "Maintenance"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:256
msgid "Online"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:258
msgid "Power State"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:259
msgid "OS Name"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:260
msgid "OS Version"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:287
msgid "Yes"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:289
msgid "No"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:312 lib/fb_vmware/app/get_storage_cluster_list.py:320
#: lib/fb_vmware/app/get_vm_list.py:350
msgid "Line template: {}"
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:328
msgid "Found no VMWare hosts."
msgstr ""

#: lib/fb_vmware/app/get_host_list.py:330
msgid "Found one VMWare host."
msgid_plural "Found {} VMWare hosts."
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:60
msgid "Tries to get a list of all datastore clusters in VMWare VSphere and print it out."
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:107
msgid "Don't print the totals of all storage clusters."
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:180
msgid "Getting all VSPhere storage clusters ..."
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:244 lib/fb_vmware/app/get_vm_info.py:213
msgid "Total"
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:285
msgid "Capacity in GB"
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:286
msgid "Free space in GB"
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:287
msgid "Calculated usage in GB"
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:288
msgid "Usage in percent"
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:349
msgid "Found one VMWare storage cluster."
msgid_plural "Found {} VMWare storage clusters."
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:352
msgid "No VMWare storage clusters found."
msgstr ""

#: lib/fb_vmware/app/get_vm_info.py:48
msgid "Tries to get information about the given virtual machines in VMWare VSphere and print it out."
msgstr ""

#: lib/fb_vmware/app/get_vm_info.py:68
msgid "Names of the VM to get information."
msgstr ""

#: lib/fb_vmware/app/get_vm_info.py:122
msgid "NOT FOUND"
msgstr ""

#: lib/fb_vmware/app/get_vm_info.py:150 lib/fb_vmware/app/get_vm_info.py:176
#: lib/fb_vmware/app/get_vm_info.py:229
msgid "Unknown"
msgstr ""

#: lib/fb_vmware/app/get_vm_info.py:193 lib/fb_vmware/app/get_vm_info.py:220
msgid "None"
msgstr ""

#: lib/fb_vmware/app/get_vm_info.py:202
msgid "Disk"
msgid_plural "Disks"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/app/get_vm_info.py:244
msgid "Custom Value"
msgid_plural "Custom Values"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/app/get_vm_info.py:265
msgid "Pulling full data of VM {!r} ..."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:59
msgid "Tries to get a list of all virtual machines in VMWare VSphere and print it out."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:136
msgid "for names of VMs"
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:137
msgid "A regular expression to filter the output list of VMs by their name (Default: {!r})."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:144
msgid "TYPE"
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:145
msgid "Filter output for the type of the VM. Valid values are {li} (Default: {dflt!r})."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:154
msgid "Filter output for online VMs."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:158
msgid "Filter output for offline VMs and templates."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:163
msgid "for VMWare hardware config version"
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:164
msgid ""
"A regular expression to filter the output list of VMs by the VMWare hardware configuration "
"version (e.g. '{}')."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:171
msgid "for the Operating System version"
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:172
msgid ""
"A regular expression to filter the output list of VMs by their Operating System version, e.g. "
"'{}'."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:181
msgid "Detailed output list (quering data needs some time longer)."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:214
msgid "Detailed output is required because of your given options."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:226
msgid "Sorting key {!r} not usable, if not detailed output was given."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:230
msgid "No usable sorting keys found, using default sorting keys {}."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:269
msgid "Getting all VSPhere VMs ..."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:279
msgid "Using sorting keys:"
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:367
msgid "Found no VMWare VMs."
msgstr ""

#: lib/fb_vmware/app/get_vm_list.py:369
msgid "Found one VMWare VM."
msgid_plural "Found {} VMWare VMs."
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/app/get_vm_list.py:436
msgid "Found a {} object:"
msgstr ""

#: lib/fb_vmware/base.py:82
msgid "The given parameter {pc!r} ({pv!r}) is not a {o} object."
msgstr ""

#: lib/fb_vmware/base.py:87
msgid "No VSPhere host name or address given in {w}."
msgstr ""

#: lib/fb_vmware/base.py:91
msgid "The {c} object given as {w} is not initialized."
msgstr ""

#: lib/fb_vmware/base.py:187
msgid "Connecting to vSphere {!r} ..."
msgstr ""

#: lib/fb_vmware/base.py:223
msgid "Please enter the user name for logging in to {}:"
msgstr ""

#: lib/fb_vmware/base.py:229
msgid "Got a {}"
msgstr ""

#: lib/fb_vmware/base.py:244
msgid "Please enter the password for {user!r} for logging in to {url}:"
msgstr ""

#: lib/fb_vmware/base.py:249
msgid "Please repeat the password for {user!r} for logging in to {url}:"
msgstr ""

#: lib/fb_vmware/base.py:266
msgid "Disconnecting from VSPhere {!r}."
msgstr ""

#: lib/fb_vmware/cluster.py:261 lib/fb_vmware/controller.py:214 lib/fb_vmware/datastore.py:356
#: lib/fb_vmware/dc.py:146 lib/fb_vmware/disk.py:275 lib/fb_vmware/ds_cluster.py:193
#: lib/fb_vmware/ether.py:315 lib/fb_vmware/host.py:546 lib/fb_vmware/host_port_group.py:161
#: lib/fb_vmware/network.py:212 lib/fb_vmware/obj.py:275 lib/fb_vmware/vm.py:469
msgid "Comparing {} objects ..."
msgstr ""

#: lib/fb_vmware/cluster.py:312
msgid "Cluster {c!r} has network {n!r}."
msgstr ""

#: lib/fb_vmware/cluster.py:319
msgid "Cluster {c!r} has datastore {d!r}."
msgstr ""

#: lib/fb_vmware/cluster.py:351 lib/fb_vmware/cluster.py:358
msgid "The given parameter {p!r} on calling method {m}() is not a sequence type."
msgstr ""

#: lib/fb_vmware/config/__init__.py:166
msgid "An empty name for a VSPhere datacenter is not allowed."
msgstr ""

#: lib/fb_vmware/config/__init__.py:314
msgid "Unknown key {k!r} with value {v!r} for VSphere {vs!r} in section {sec!r} found."
msgstr ""

#: lib/fb_vmware/config/__init__.py:320
msgid "{e} in section {sn!r} for VSphere {vs!r}:"
msgstr ""

#: lib/fb_vmware/config/__init__.py:326
msgid "There must be given at least the VSPhere hostname in section {sn!r} for VSphere {vs!r}."
msgstr ""

#: lib/fb_vmware/config/__init__.py:469
msgid "Evaluating config section {!r}:"
msgstr ""

#: lib/fb_vmware/config/__init__.py:478
msgid "Empty VSphere name found."
msgstr ""

#: lib/fb_vmware/connect.py:120
msgid "Trying to get some 'about' information from VSphere."
msgstr ""

#: lib/fb_vmware/connect.py:135
msgid "Got a {c} on requesting 'about' information from VSPhere {url}: {e}"
msgstr ""

#: lib/fb_vmware/connect.py:145
msgid "VSphere version: {!r}"
msgstr ""

#: lib/fb_vmware/connect.py:147
msgid "Found VSphere about-information:"
msgstr ""

#: lib/fb_vmware/connect.py:152
msgid "Trying to get datacenter from VSphere ..."
msgstr ""

#: lib/fb_vmware/connect.py:166
msgid "Found VSphere datacenter {!r}."
msgstr ""

#: lib/fb_vmware/connect.py:168
msgid "Info about datacenter:"
msgstr ""

#: lib/fb_vmware/connect.py:179
msgid "Trying to get all clusters from VSphere ..."
msgstr ""

#: lib/fb_vmware/connect.py:203 lib/fb_vmware/connect.py:208
msgid "Found clusters:"
msgstr ""

#: lib/fb_vmware/connect.py:224 lib/fb_vmware/connect.py:492
msgid "Found standalone host"
msgstr ""

#: lib/fb_vmware/connect.py:226 lib/fb_vmware/connect.py:494
msgid "Found cluster"
msgstr ""

#: lib/fb_vmware/connect.py:227 lib/fb_vmware/connect.py:495
msgid "host"
msgid_plural "hosts"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:228 lib/fb_vmware/connect.py:496
msgid "CPU"
msgid_plural "CPUs"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:229 lib/fb_vmware/connect.py:497
msgid "thread"
msgid_plural "threads"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:230 lib/fb_vmware/connect.py:498
msgid "network"
msgid_plural "networks"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:231 lib/fb_vmware/connect.py:499
msgid "datastore"
msgid_plural "datastores"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:232 lib/fb_vmware/connect.py:500
msgid ""
"{on} {cl!r}, {h} {h_l}, {cpu} {cpu_l}, {thr} {t_l}, {mem:0.1f} GiB Memory, {net} {nw_l} and {ds}"
" {ds_l}."
msgstr ""

#: lib/fb_vmware/connect.py:255
msgid "Trying to get all datastores from VSphere ..."
msgstr ""

#: lib/fb_vmware/connect.py:278 lib/fb_vmware/connect.py:280
msgid "Found datastores:"
msgstr ""

#: lib/fb_vmware/connect.py:288
msgid "Datastore mappings:"
msgstr ""

#: lib/fb_vmware/connect.py:303
msgid "Datastore {!r} seems to be local."
msgstr ""

#: lib/fb_vmware/connect.py:309
msgid "Found datastore {ds!r} of type {t!r}, capacity {c:0.1f} GByte."
msgstr ""

#: lib/fb_vmware/connect.py:318
msgid "Trying to get all datastore clusters from VSphere ..."
msgstr ""

#: lib/fb_vmware/app/get_storage_cluster_list.py:189 lib/fb_vmware/connect.py:342
#: lib/fb_vmware/connect.py:345
msgid "Found datastore clusters:"
msgstr ""

#: lib/fb_vmware/connect.py:347
msgid "No VSphere datastore clusters found."
msgstr ""

#: lib/fb_vmware/connect.py:353
msgid "Datastore cluster mappings:"
msgstr ""

#: lib/fb_vmware/connect.py:359 lib/fb_vmware/connect.py:419
msgid "Found a {} child."
msgstr ""

#: lib/fb_vmware/connect.py:377
msgid "Trying to get all networks from VSphere ..."
msgstr ""

#: lib/fb_vmware/connect.py:399
msgid "Found one VSphere network."
msgid_plural "Found {n} VSphere networks."
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:403 lib/fb_vmware/connect.py:405
msgid "Found VSphere networks:"
msgstr ""

#: lib/fb_vmware/connect.py:407
msgid "No VSphere networks found."
msgstr ""

#: lib/fb_vmware/connect.py:413
msgid "Network mappings:"
msgstr ""

#: lib/fb_vmware/connect.py:439 lib/fb_vmware/connect.py:643
msgid "Parameter {p!r} => {r!r} seems not to be a regex object."
msgstr ""

#: lib/fb_vmware/connect.py:442
msgid "Trying to get all host systems from VSphere with name pattern {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:446
msgid "Trying to get all host systems from VSphere ..."
msgstr ""

#: lib/fb_vmware/connect.py:473 lib/fb_vmware/connect.py:478
msgid "Found hosts:"
msgstr ""

#: lib/fb_vmware/connect.py:484
msgid "Checking {o}-object in cluster {c!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:518
msgid "Found host {h!r} in cluster {c!r}."
msgstr ""

#: lib/fb_vmware/connect.py:533
msgid "Searching for VM {n!r} (pattern: {p!r}) in VSPhere {v!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:541
msgid "VSphere VM {!r} not found."
msgstr ""

#: lib/fb_vmware/connect.py:554 lib/fb_vmware/datastore.py:239 lib/fb_vmware/ds_cluster.py:138
#: lib/fb_vmware/iface.py:59 lib/fb_vmware/network.py:149
msgid "Parameter {t!r} must be a {e}, {v!r} was given."
msgstr ""

#: lib/fb_vmware/connect.py:647
msgid "Parameter {p1!r} and {p2!r} may not be {w!r} at the same time."
msgstr ""

#: lib/fb_vmware/connect.py:651
msgid "Trying to get list of VMs with name pattern {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:671
msgid "Searching in path {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:684
msgid "Found one VM with pattern {p!r}."
msgid_plural "Found {no} VMs with pattern {p!r}."
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:719
msgid "Checking VM {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:722
msgid "Checking VM {!r} for being a template ..."
msgstr ""

#: lib/fb_vmware/connect.py:724
msgid "Checking VM {!r} for being not a template ..."
msgstr ""

#: lib/fb_vmware/connect.py:732
msgid "Checking VM {!r} for pattern."
msgstr ""

#: lib/fb_vmware/connect.py:735
msgid "Found VM {!r}."
msgstr ""

#: lib/fb_vmware/connect.py:799
msgid "VM {!r} is already powered on."
msgstr ""

#: lib/fb_vmware/connect.py:802
msgid "Powering on VM {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:806
msgid "VM {!r} successful powered on."
msgstr ""

#: lib/fb_vmware/connect.py:830
msgid "VM {!r} is already powered off."
msgstr ""

#: lib/fb_vmware/connect.py:833
msgid "Powering off VM {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:837
msgid "VM {!r} successful powered off."
msgstr ""

#: lib/fb_vmware/connect.py:846
msgid "Ensuring existence of VSphere VM folders:"
msgstr ""

#: lib/fb_vmware/connect.py:863
msgid "Trying to get VM folder object for path {!r}."
msgstr ""

#: lib/fb_vmware/connect.py:888
msgid "Checking single VM folder {i}: {f!r}."
msgstr ""

#: lib/fb_vmware/connect.py:899
msgid "Found VM folder {n}, parent: {p}"
msgstr ""

#: lib/fb_vmware/connect.py:917
msgid "Ensuring existence of VSphere VM folder {!r}."
msgstr ""

#: lib/fb_vmware/connect.py:942
msgid "VM Folder {!r} already exists."
msgstr ""

#: lib/fb_vmware/connect.py:944
msgid "Creating VM folder {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:946
msgid "Simulation mode, don't creating it."
msgstr ""

#: lib/fb_vmware/connect.py:962
msgid "Waiting for tasks to finish ..."
msgstr ""

#: lib/fb_vmware/connect.py:974
msgid "Waiting at most {m} seconds for tasks {t} to finish ..."
msgstr ""

#: lib/fb_vmware/connect.py:977
msgid "Waiting for tasks {} to finish ..."
msgstr ""

#: lib/fb_vmware/connect.py:997
msgid "Waiting ..."
msgstr ""

#: lib/fb_vmware/connect.py:1031
msgid "Creating VM {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:1034
msgid "Simulation mode - VM {!r} will not be created."
msgstr ""

#: lib/fb_vmware/connect.py:1053
msgid "Generating create spec for VM {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:1061
msgid "Datastore path: {!r}"
msgstr ""

#: lib/fb_vmware/connect.py:1064
msgid "VM path name: {!r}"
msgstr ""

#: lib/fb_vmware/connect.py:1121
msgid "Generated VM config:"
msgstr ""

#: lib/fb_vmware/connect.py:1130
msgid "Given disksize {!r} must be greater than zero."
msgstr ""

#: lib/fb_vmware/connect.py:1143
msgid "There may be created at most 6 disks, but {} were given."
msgstr ""

#: lib/fb_vmware/connect.py:1157
msgid "Generating spec for SCSI controller and one disk: {d}"
msgid_plural "Generating spec for SCSI controller and {n} disks: {d}"
msgstr[0] ""
msgstr[1] ""

#: lib/fb_vmware/connect.py:1160
msgid "Generating spec for SCSI controller without disks."
msgstr ""

#: lib/fb_vmware/connect.py:1167
msgid "Using a {name!r} disk controller ({desc})."
msgstr ""

#: lib/fb_vmware/connect.py:1190
msgid "Adding spec for disk {n!r} with {gb} GiB => {kb} KiByte."
msgstr ""

#: lib/fb_vmware/connect.py:1225
msgid "Invalid Interface description {!r} given."
msgstr ""

#: lib/fb_vmware/connect.py:1236
msgid "Defined interface:"
msgstr ""

#: lib/fb_vmware/connect.py:1240
msgid "Adding spec for network interface {d!r} (Network {n!r}, MAC: {m!r}, summary: {s!r})."
msgstr ""

#: lib/fb_vmware/connect.py:1292
msgid "Purging VM {!r} ..."
msgstr ""

#: lib/fb_vmware/connect.py:1296
msgid "VM {!r} successful removed."
msgstr ""

#: lib/fb_vmware/connect.py:1327
msgid "Did not found virtual ethernet device No. {no} (found {count} devices)."
msgstr ""

#: lib/fb_vmware/connect.py:1346
msgid "Changes of MAC address:"
msgstr ""

#: lib/fb_vmware/connect.py:1350
msgid "Successful changed MAC address of VM {v!r} to {m!r}."
msgstr ""

#: lib/fb_vmware/connect.py:1365
msgid "Trying to detect all field names of custom field definitions."
msgstr ""

#: lib/fb_vmware/connect.py:1382
msgid "Got a {c} on requesting custom field names from VSPhere {url}: {e}"
msgstr ""

#: lib/fb_vmware/connect.py:1388
msgid "Got custom field names from VSPhere {}:"
msgstr ""

#: lib/fb_vmware/controller.py:58
msgid "Virtual IDE controller"
msgstr ""

#: lib/fb_vmware/controller.py:59
msgid "Virtual VNME controller"
msgstr ""

#: lib/fb_vmware/controller.py:60
msgid "Virtual PCI controller"
msgstr ""

#: lib/fb_vmware/controller.py:61
msgid "Virtual controller for keyboards and mice"
msgstr ""

#: lib/fb_vmware/controller.py:62
msgid "Virtual AHCI SATA controller"
msgstr ""

#: lib/fb_vmware/controller.py:63
msgid "Virtual SATA controller"
msgstr ""

#: lib/fb_vmware/controller.py:64
msgid "Virtual paravirtualized SCSI controller"
msgstr ""

#: lib/fb_vmware/controller.py:65
msgid "Virtual BusLogic SCSI controller"
msgstr ""

#: lib/fb_vmware/controller.py:66
msgid "Virtual LSI SCSI controller"
msgstr ""

#: lib/fb_vmware/controller.py:67
msgid "Virtual LSI Logic SAS SCSI controller"
msgstr ""

#: lib/fb_vmware/controller.py:68
msgid "Virtual SCSI controller"
msgstr ""

#: lib/fb_vmware/controller.py:69
msgid "Virtual Super IO Controller for floppy drives, parallel ports, and serial ports"
msgstr ""

#: lib/fb_vmware/controller.py:71
msgid "Virtual USB controller (USB 1.1 and 2.0)"
msgstr ""

#: lib/fb_vmware/controller.py:72
msgid "Virtual USB Extensible Host Controller Interface (USB 3.0)"
msgstr ""

#: lib/fb_vmware/controller.py:73
msgid "Unknown virtual controller"
msgstr ""

#: lib/fb_vmware/controller.py:301 lib/fb_vmware/ether.py:424 lib/fb_vmware/host.py:255
#: lib/fb_vmware/host.py:568 lib/fb_vmware/host_port_group.py:206 lib/fb_vmware/vm.py:493
msgid "Parameter {t!r} must be a {e}, {v!r} ({vt}) was given."
msgstr ""

#: lib/fb_vmware/controller.py:323
msgid "Checking class of controller: {!r}"
msgstr ""

#: lib/fb_vmware/controller.py:348 lib/fb_vmware/disk.py:433 lib/fb_vmware/ether.py:521
#: lib/fb_vmware/host_port_group.py:236 lib/fb_vmware/vm.py:651
msgid "Invalid type {t!r} as an item of a {c}, only {o} objects are allowed."
msgstr ""

#: lib/fb_vmware/controller.py:415 lib/fb_vmware/disk.py:500 lib/fb_vmware/ether.py:588
#: lib/fb_vmware/host.py:792 lib/fb_vmware/host_port_group.py:324 lib/fb_vmware/vm.py:718
msgid "{m} takes at most {max} arguments ({n} given)."
msgstr ""

#: lib/fb_vmware/controller.py:454
msgid "Controller is not in controller list."
msgstr ""

#: lib/fb_vmware/datastore.py:375 lib/fb_vmware/ds_cluster.py:212 lib/fb_vmware/network.py:231
msgid "Invalid item type {{!r}} to set, only {} allowed."
msgstr ""

#: lib/fb_vmware/datastore.py:377
msgid "The key {k!r} must be equal to the datastore name {n!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:378 lib/fb_vmware/ds_cluster.py:215 lib/fb_vmware/network.py:234
msgid "None type as key is not allowed."
msgstr ""

#: lib/fb_vmware/datastore.py:379 lib/fb_vmware/ds_cluster.py:216 lib/fb_vmware/network.py:235
msgid "Empty key {!r} is not allowed."
msgstr ""

#: lib/fb_vmware/datastore.py:380 lib/fb_vmware/ds_cluster.py:217 lib/fb_vmware/network.py:236
#: lib/fb_vmware/obj.py:257 lib/fb_vmware/obj.py:266 lib/fb_vmware/obj.py:286 lib/fb_vmware/obj.py:298
msgid "Object {{!r}} is not a {} object."
msgstr ""

#: lib/fb_vmware/datastore.py:623
msgid "Could not handle datastore type {!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:630
msgid "Could not found a datastore for {c:0.1f} GiB of type {t!r}."
msgstr ""

#: lib/fb_vmware/datastore.py:637
msgid "Searching datastore for {c:0.1f} GiB of type {t!r}."
msgstr ""

#: lib/fb_vmware/disk.py:540
msgid "Disk is not in disk list."
msgstr ""

#: lib/fb_vmware/ds_cluster.py:214
msgid "The key {k!r} must be equal to the datastore cluster name {n!r}."
msgstr ""

#: lib/fb_vmware/errors.py:55
msgid "No VSphere datastores found."
msgstr ""

#: lib/fb_vmware/errors.py:89
msgid "Invalid credentials to connect to Vsphere as user {!r}: no password given."
msgstr ""

#: lib/fb_vmware/errors.py:93
msgid "Invalid credentials to connect to Vsphere: no user given."
msgstr ""

#: lib/fb_vmware/errors.py:109
msgid "The given disk controller type {!r} could not be found."
msgstr ""

#: lib/fb_vmware/errors.py:127
msgid "Invalid name {n!r} for a {o} VSphere object."
msgstr ""

#: lib/fb_vmware/errors.py:130
msgid "Invalid name {!r} for a VSphere object."
msgstr ""

#: lib/fb_vmware/errors.py:147
msgid "The VSphere datacenter {!r} is not existing."
msgstr ""

#: lib/fb_vmware/errors.py:163
msgid "The VSphere Virtual machine {!r} was not found."
msgstr ""

#: lib/fb_vmware/errors.py:182
msgid "No SAN based datastore found with at least {m:0.0f} MiB == {g:0.1f} GiB available space found."
msgstr ""

#: lib/fb_vmware/errors.py:200
msgid "The network {!r} is not existing."
msgstr ""

#: lib/fb_vmware/errors.py:216
msgid "Could not connect to the vSphere {!r}."
msgstr ""

#: lib/fb_vmware/errors.py:232
msgid "Got a {c} on connecting to vSphere {url!r}:"
msgstr ""

#: lib/fb_vmware/errors.py:259
msgid "Timeout on creating VM {vm!r} after {to:0.1f} seconds."
msgstr ""

#: lib/fb_vmware/errors.py:262
msgid "Timeout on creating VM {!r}."
msgstr ""

#: lib/fb_vmware/errors.py:279
msgid "Invalid type of {!r} for a port of a VSPhere server"
msgstr ""

#: lib/fb_vmware/errors.py:304
msgid ""
"Invalid port number {port!r} for the VSphere server, PORT must be greater than zero and less or "
"equal to {max}."
msgstr ""

#: lib/fb_vmware/ether.py:452
msgid "Checking class of ethernet card: {!r}"
msgstr ""

#: lib/fb_vmware/ether.py:627
msgid "card is not in card list."
msgstr ""

#: lib/fb_vmware/host.py:346 lib/fb_vmware/vm.py:110
msgid "The name of the vsphere may not be empty."
msgstr ""

#: lib/fb_vmware/host.py:573
msgid "Host {!r} seems to be offline!"
msgstr ""

#: lib/fb_vmware/host.py:723
msgid "Invalid type {{t!r}} as an item of a {{c}}, only {} objects are allowed."
msgstr ""

#: lib/fb_vmware/host.py:831
msgid "host is not in host list."
msgstr ""

#: lib/fb_vmware/host_port_group.py:363
msgid "group is not in group list."
msgstr ""

#: lib/fb_vmware/iface.py:127
msgid "Invalid MAC address {!r} for interface given."
msgstr ""

#: lib/fb_vmware/network.py:72
msgid "Trying to get IPv4 network {n!r} -> {i!r}."
msgstr ""

#: lib/fb_vmware/network.py:79
msgid "Could not get IP network from network name {!r}."
msgstr ""

#: lib/fb_vmware/network.py:82
msgid "Network {!r} has no IP network assigned."
msgstr ""

#: lib/fb_vmware/network.py:88
msgid "Initialized network object:"
msgstr ""

#: lib/fb_vmware/network.py:233
msgid "The key {k!r} must be equal to the network name {n!r}."
msgstr ""

#: lib/fb_vmware/network.py:501
msgid "Searching VSphere network for address {} ..."
msgstr ""

#: lib/fb_vmware/network.py:507
msgid "Found network {n!r} for IP {i}."
msgstr ""

#: lib/fb_vmware/network.py:511
msgid "Could not find VSphere network for IP {}."
msgstr ""

#: lib/fb_vmware/network.py:514
msgid "Could not find VSphere network for IP addresses {}."
msgstr ""

#: lib/fb_vmware/obj.py:84
msgid "The type of a {} may not be None."
msgstr ""

#: lib/fb_vmware/obj.py:89 lib/fb_vmware/obj.py:112 lib/fb_vmware/obj.py:135
msgid "Invalid {w}.{p} {v!r}."
msgstr ""

#: lib/fb_vmware/obj.py:151
msgid "The name prefix of a {} may not be None."
msgstr ""

#: lib/fb_vmware/obj.py:155
msgid "Invalid name prefix {p!r} for a {o}."
msgstr ""

#: lib/fb_vmware/vm.py:582
msgid "Unknown hardware device of type {}."
msgstr ""

#: lib/fb_vmware/vm.py:585
msgid "There is something wrong wit VM {n!r} in cluster {c!r} and path {p!r} ..."
msgstr ""

#: lib/fb_vmware/vm.py:757
msgid "VM is not in VM list."
msgstr ""

#: lib/fb_vmware/xlate.py:123
msgid "Module directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:124
msgid "Lib directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:125
msgid "Base directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:126
msgid "Locale directory:"
msgstr ""

#: lib/fb_vmware/xlate.py:127
msgid "Locale domain:"
msgstr ""

#: lib/fb_vmware/xlate.py:128
msgid "Found .mo-file:"
msgstr ""

